src/tools/helpers.js

```diff
@@ -49,3 +49,12 @@ export const getIdsText = (ids) => {
   }
   return string;
 };
+
+// convert vw and vh dimensions to px
+export const viewportToPixels = (value) => {
+  const parts = value.match(/([0-9.]+)(vh|vw)/);
+  const q = Number(parts[1]);
```

Comment:
what is `q`? Can we name this better?

Reply:
Yep, it's the units. So, for '70vh', it'd be 70. I renamed it to units. Good catch though, I didn't actually ever look at how this code worked until now.